lib/releases.sh

```diff
@@ -58,9 +58,14 @@ elif [ "${CAPM3RELEASEBRANCH}" == "release-1.1" ]; then
 elif [ "${CAPM3RELEASEBRANCH}" == "release-1.2" ]; then
   export CAPM3RELEASE="${CAPM3RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v1.2.")}"
   export CAPIRELEASE="${CAPIRELEASE:-$(get_latest_release "${CAPIRELEASEPATH}" "v1.2.")}"
-else
+elif [ "${CAPM3RELEASEBRANCH}" == "release-1.3" ]; then
```

Comment:
Let's not add any  new `[` and `==` combinations even if the previous ones have it.

Reply:
is the change ok now @tuminoid ?